From 3a3f1b317328c8cae353c01d6f4e6523be0fe3cb Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 5 Jul 2023 17:54:10 +0200
Subject: [PATCH 0597/1695] plat-stm32mp2: conf: enable PWR interrupts

Enable PWR interrupts on STM32MP2.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 99000f90b06208d4ea934534e53db18dd8601276 plat-stm32mp2: conf: enable PWR interrupts]
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: I8d87bcb251552e905b0a4ed946fd581104b67d1f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/319651
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index c8fe52edc..c1ce4fbc0 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -96,6 +96,7 @@ CFG_STM32_IAC ?= y
 CFG_STM32_IPCC ?= y
 CFG_STM32_IWDG ?= y
 CFG_STM32_OMM ?= y
+CFG_STM32_PWR_IRQ ?= y
 CFG_STM32_RIF ?= y
 CFG_STM32_RIFSC ?= y
 CFG_STM32_RISAB ?= y
-- 
2.47.2

